definition/data/sheets/ComplexTypes.json

```diff
@@ -221,6 +221,7 @@
   {"ID": "Respondent", "ListElementCode": "claimantProvidedRepresentativeOrganisationDxAddress", "FieldType": "Text", "ElementLabel": "Organisation DxAddress", "SecurityClassification": "Public"},
   {"ID": "Respondent", "ListElementCode": "claimantProvidedRepresentativeOrganisationEmail", "FieldType": "Email", "ElementLabel": "Organisation Email", "SecurityClassification": "Public"},
   {"ID": "Respondent", "ListElementCode": "directionsQuestionnaire", "FieldType": "DirectionsQuestionnaire", "ElementLabel": "Respondent Directions Questionnaire", "SecurityClassification": "Public"},
+  {"ID": "Respondent", "ListElementCode": "responseMethod", "FieldType": "FixedList",  "FieldTypeParameter": "ResponseMethod", "ElementLabel": "Defendants Response Method", "SecurityClassification": "Public"},
```

Comment:
Shouldn't it be "Defendant's Response Method"?

Reply:
Updated